pkg/mapper/entity_restapi/r2e/r2e.go

```diff
@@ -36,15 +36,9 @@ func MapAttachment(a interface{}) (entity.Attachment, error) {
 	if a != nil {
 		m, ok := a.(map[string]interface{})
 		if !ok {
-			return e, fmt.Errorf("all key/value pairs should be string/string. invalid attachment format %s", spew.Sdump(a))
-		}
-		for k, v := range m {
-			s, ok := v.(string)
-			if !ok {
-				return e, fmt.Errorf("all key/value pairs should be string/string. invalid attachment format %s", spew.Sdump(a))
-			}
-			e[k] = s
+			return e, fmt.Errorf("Make sure JSON is properly formatted. Invalid attachment format %s", spew.Sdump(a))
```

Comment:
one last nit, JSON format actually supports things like naked string, number, or null. here we may want to say:

"Make sure JSON is properly formatted into key/value pairs."

Reply:
No problem, pushing a fix now if you want to take a quick look